File: articles/norbert-wiener-cybernetics-and-feedback-in-computation/typescript/negative_feedback.ts
let x = 10.0;
const target = 0.0;
const gain = 0.2;
for (let i = 0; i < 5; i += 1) {
  x = x - gain * (x - target);
}
console.log(`final_state=${x.toFixed(6)}`);

File: articles/norbert-wiener-cybernetics-and-feedback-in-computation/go/negative_feedback.go
package main

import "fmt"

func main() {
	x := 10.0
	target := 0.0
	gain := 0.2
	for i := 0; i < 5; i++ {
		x = x - gain*(x-target)
	}
	fmt.Printf("final_state=%.6f\n", x)
}
